zuul.d/adoption.yaml

```diff
@@ -114,3 +114,151 @@
       - LICENSE
       - OWNERS
       - .*/*.md
+
+- job:
+    name: cifmw-adoption-base-source-multinode
+    parent: base-extracted-crc
+    abstract: true
+    timeout: 10800
+    attempts: 1
+    nodeset:
```

Comment:
yes @lewisdenny absolutely
the reason i didn't is because the definition is really not done yet and i was going to say 'yes will do later' 

but thinking about it now i can still iterate on it there instead of here so..  doing ;)